Gemfile

```diff
@@ -47,3 +47,6 @@ end
 gem 'tzinfo-data', platforms: [:mingw, :mswin, :x64_mingw, :jruby]
 
 gem 'sendgrid-ruby'
+
+gem 'pundit'
```

Comment:
Remove this. There's a saying for remembering not to get ahead of yourself: ["You ain't gonna need it"](https://en.wikipedia.org/wiki/You_aren%27t_gonna_need_it).